src/benchmarking/config.yml

```
@@ -0,0 +1,38 @@
+# you can ignore this for now
+datasets:
+  - name: 'Dataset'
+    # not used for now, but could be useful in future
+    # file: 'some-dataset-file.nt'
+
+#Your connection
+connections:
+  - name: 'Virtuoso'
+    endpoint: 'http://ec2-18-170-45-162.eu-west-2.compute.amazonaws.com:8890/sparql'
+  - name: 'Neptune'
+    endpoint: 'https://geonames-rdf-01.cluster-ro-ci9272ypsbyf.eu-west-2.neptune.amazonaws.com:8182/sparql'
+
+tasks:
+  - className: "Stresstest"
+    configuration:
+      #timeLimit is in ms
+      timeLimit: 3600000
```

Minor, this could probably be:
```suggestion
      timeLimit: 3600000 # ms
```